### DISCUSSION TITLE
HowTo Ubuntu 22.04 desktop zoom

### DISCUSSION THREAD

**Discussion by mrspirytus** [sezanzeb/input-remapper#696] [Q&A]
I just installed input-mapper 2.0 on Ubuntu 22.04. I was able to configure extra mouse buttons to Super_L and other to Super_L + KEY_A. This is working well.
What I like to do is activate desktop zoon when pressing ALT key and while holding it down, scroll with mouse wheel to zoom in and out.
I already did enable Zoom in Settings / Accessibility / Zoom Options. I can zoom in and out using builtin keyboard shortcuts
Super Alt + and Super Alt -
I am not 100% set on using ALT key for wheel activation if that conflicts with something already.
In addition, if there is a way to control zoom step vs the default zoom steps.
Thank you in advance.

**Comment by jonasBoss** [comment]
This requires the combination of multiple input devices (keyboard + mouse) into one mapping. Unfortunately this is currently not well supported due to the inner workings of InputRemapper. It might be possible using macro variables. If I ever get around to finish #592 this could be implemented

It is possible to create this with just the mouse. You can use any button on you mouse plus the scroll wheel.
Create one mapping for each direction (using button left in this example):
 - `Button LEFT + Wheel Down + Wheel HI RES -` -> `Super_L + Alt_L + minus`
 - `Button LEFT + Wheel Up + Wheel HI RES +` -> `Super_L + Alt_L + plus`

**Reply by oxoocoffee** [reply]
I love this feature on MacOX. Works great … mouse wheel feels so much more natural if you are using it a lot (eyesight). Looking forward one day for this update :)

**Reply by jonasBoss** [reply]
as I said, you can do the mapping already with the scroll wheel and a mouse button.

**Reply by oxoocoffee** [reply]
I was thinking, would there be an option to do some financial contribution to get this implemented? Just wondering.
